### PR TITLE
[T4A4][W11-B4]Tay Keming Justin

### PR DESCRIPTION
**Pull request by INCENDE**

### CODE REVIEW

**Comment by INCENDE** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:2]: Please include a header comment to indicate what this interface/class/method does?

File: src/seedu/addressbook/data/person/Name.java [hunk 2]
```diff
@@ -61,4 +61,9 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    @Override
+    public String getPrintableString() {
+	return getClass().getCanonicalName() + ": " + this.toString();
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Name.java:66]: this is good! but think which is more explicit and simpler to read using the word "Email" or `getClass().getCanonicalName()`

**Comment by okkhoy** [comment]
@INCENDE 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**